### ISSUE TITLE
Instead "Connecting to OctoPrint" a Homescreen

### ISSUE THREAD

**Issue by xeenon86** [mcuadros/OctoPrint-TFT#53]
If the Pi isnt connected to the Printer Board I only see the connecting screen.

My printer is powered by a switchable PSU which is controlled by a Tasmota Wlan Power switch. After a Print job the printer is shutting down. The raspberry is always on. 

 Is it possible to show a Homescreen with custom buttons if the board isn't connected to the raspberry? Like a button to switch the psu on?

**Comment by Enverex** [comment]
Yeah, this is incredibly confusing. I was wondering why it wasn't connecting despite all the details being right. The issue infact being that it's just simply not connected to a printer right now.

**Comment by dentex** [comment]
I was asking myself the same thing. I would like to have the UI active and manage myself (as fallback to automatic) the connection to the printer.
This way, you can control the printer to power on and off i.e. via a relay board.
I'm actually using the fork at https://github.com/darksid3r/OctoPrint-TFT , but there are no "issues" there.
